### ISSUE TITLE
[expo-sqlite] Drizzle ORM not compatible with expo-sqlite 15

### ISSUE THREAD

**Issue by tomnoir** [expo/expo#32714]
### Minimal reproducible example

https://github.com/tomnoir/expo-52-drizzle-issue-example

### What platform(s) does this occur on?

Android, iOS, Web

### Where did you reproduce the issue?

in Expo Go

### Summary

Drizzle ORM has still exports from expo-sqlite/next, which was removed with v15/SDK 52.
There is already a PR with Drizzle, but it has not been merged yet: https://github.com/drizzle-team/drizzle-orm/pull/3197
This is of course no issue with Expo/Expo SQLite, but wanted to make the issue aware before the release of SDK 52. 

### Environment

```text
expo-env-info 1.2.0 environment info:
    System:
      OS: macOS 15.0.1
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 20.18.0 - /opt/homebrew/opt/node@20/bin/node
      Yarn: 1.22.21 - /opt/homebrew/bin/yarn
      npm: 10.8.2 - /opt/homebrew/opt/node@20/bin/npm
      Watchman: 2024.10.28.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.15.2 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 24.1, iOS 18.1, macOS 15.1, tvOS 18.1, visionOS 2.1, watchOS 11.1
      Android SDK:
        API Levels: 31, 32, 33, 34, 35
        Build Tools: 33.0.0, 33.0.1, 34.0.0, 35.0.0
        System Images: android-30 | Google APIs ARM 64 v8a, android-31 | Google APIs ARM 64 v8a, android-31 | Google Play ARM 64 v8a, android-34 | Desktop ARM 64 v8a, android-34 | ARM 64 v8a
    IDEs:
      Android Studio: 2023.1 AI-231.9392.1.2311.11330709
      Xcode: 16.1/16B40 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~52.0.0-preview.23 => 52.0.0-preview.23 
      expo-router: ~4.0.0-preview.14 => 4.0.0-preview.14 
      react: 19.0.0-rc-5c56b873-20241107 => 19.0.0-rc-5c56b873-20241107 
      react-dom: 19.0.0-rc-5c56b873-20241107 => 19.0.0-rc-5c56b873-20241107 
      react-native: 0.76.1 => 0.76.1 
      react-native-web: ~0.19.13 => 0.19.13 
    npmGlobalPackages:
      eas-cli: 13.1.0
    Expo Workflow: managed
```

### Expo Doctor Diagnostics

```text
Enabled experimental React Native Directory checks. Unset the EXPO_DOCTOR_ENABLE_DIRECTORY_CHECK environment variable to disable this check.
✔ Check package.json for common issues
✔ Check Expo config for common issues
✔ Check native tooling versions
✔ Check if the project meets version requirements for submission to app stores
✔ Check for common project setup issues
✔ Check dependencies for packages that should not be installed directly
✔ Check for app config fields that may not be synced in a non-CNG project
✔ Check for issues with Metro config
✔ Check npm/ yarn versions
✖ Validate packages against React Native Directory package metadata
✔ Check Expo config (app.json/ app.config.js) schema
✔ Check that packages match versions required by installed Expo SDK
✔ Check that native modules do not use incompatible support packages
✔ Check for legacy global CLI installed locally
✔ Check that native modules use compatible support package versions for installed Expo SDK

Detailed check results:

The following issues were found when validating your dependencies against React Native Directory:
No metadata available: drizzle-orm
Advice: 
- Update React Native Directory to include metadata for unknown packages. Alternatively, set expo.doctor.reactNativeDirectoryCheck.listUnknownPackages in package.json to false to skip warnings about packages with no metadata, if the warning is not relevant.

One or more checks failed, indicating possible issues with the project.
```

**Comment by brentvatne** [comment]
thanks for sharing! there isn't much we can do here, it's up to the drizzle team to land that and release it

**Comment by monkeyK1n9** [comment]
Facing the same issue, I guess I have to do a PR on drizzle

**Comment by monkeyK1n9** [comment]
> Facing the same issue, I guess I have to do a PR on drizzle

Drizzle already fixed the issue with the 0.37.0 release version.
